### DISCUSSION TITLE
Will the headless mode return in v2?

### DISCUSSION THREAD

**Discussion by einschmidt** [automadcms/discussions#22] [Q&A]
Hi,
I just stumbled across automad, and love to see the recent changes for v2.
However, one feature I am missing is the headless mode, which was removed in alpha-1.

Are there any plan to bring back the headless mode, like in "it has been disabled for the time being"?
Or is the headless mode gone forever?

I am just curios.
Reason is that a simple editor UI + headless API for content is a very nice mix and match (for my current and future projects).

**Comment by marcantondahmen** [comment]
Hey, technically there's no problem at all to bring back the headless mode. It is currently not implemented since version 2 is basically a full rewrite and I was not aware of it being used by anyone out there. But good to know there's interest in it!

**Reply by 127zeppelin** [reply]
Hi. I use headless mode on two small projects and I really like it.
